src/Common/tests/gtest_rw_lock.cpp

```diff
@@ -24,6 +24,39 @@ namespace DB
 }
 
 
+namespace
+{
+    class Events
+    {
+    public:
+        Events() : start_time(std::chrono::steady_clock::now()) {}
+
+        void add(String && event)
+        {
+            String timepoint = std::to_string(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now() - start_time).count());
+            if (timepoint.length() < 5)
+                timepoint.insert(0, 5 - timepoint.length(), ' ');
+            std::lock_guard lock{mutex};
+            std::cout << timepoint << " : " << event << std::endl;
```

Comment:
We can remove the prints now and leave the asserts

Reply:
done